src/Cryptol/Parser/Unlit.hs

```diff
@@ -8,7 +8,7 @@
 --
 -- Convert a literate source file into an ordinary source file.
 
-{-# LANGUAGE OverloadedStrings, Safe, PatternGuards #-}
+{-# LANGUAGE OverloadedStrings, PatternGuards #-}
```

Comment:
Is the code in this module no longer `Safe`?

Reply:
It is still Safe.   I think we've been using `Safe` incorrectly, probably due to a historical misunderstanding on my part about what the pragma means, so I thought I might as well start fixing things slowly.  In particular, GHC can already infer which modules are safe and which are not, without the pragma.  Instead,   the pragma explicitly asks GHC to *check* if a module is safe, and complain if it is not.  So I don't think we need to annotate every single module with `Safe`, but rather we should add it to modules where we want this checked. We currently don't really have a use case for this feature, so I am not sure which modules should have the annotation...

Reply:
Ah, I see. I don't have too strong of an opinion about `Safe` myself (if anything, I am slightly biased against it), so I'm happy to defer to your judgment about whether they should be kept or removed.